lighthouse-core/audits/audit.js

```diff
@@ -125,9 +124,9 @@ class Audit {
   }
 
   /**
-   * @param {!Audit} audit
-   * @param {!AuditResult} result
-   * @return {!AuditFullResult}
+   * @param {typeof Audit} audit
+   * @param {LH.AuditResult} result
```

Comment:
Something that's been bugging me lately.. In our external LHR docs we want to basically call `LH.AuditFullResult` the "Audit Result". So one alternative here is...

* `LH.AuditResult` => something.. (like LH.AuditReturnValue) and 
* `LH.AuditFullResult` => `LH.AuditResult`

Reply:
yeah, the problem used to be we needed a nice name in each audit and we didn't care so much what came out the other side. I don't mind the switch but we should come up with a good alternative for the first one (and maybe punt on this because there are a lot of `AuditResult`s that would need to change

Reply:
fine with punting for today.